src/app/views/components/date-range-picker/date-range-picker-reactive-form/date-range-picker-reactive-form.component.ts

```diff
@@ -12,7 +12,7 @@ export class DateRangePickerReactiveFormComponent implements OnInit {
 
   public demoForm: FormGroup;
   public dateRangeControl: FormControl;
-  public valueMode = 'object';
```

Comment:
Not sure if a newer version of TS would have better inference, but had to type this explicitly or I'd get an error in the template that type `string` is not assignable to type `DatePickerValueMode`.  The latter type is not exported from `@tylertech/forge` either.  Although I suppose in typical non-demo usage, this probably wouldn't be a binding, and just setting the attribute to the literal `valueMode="object"` works fine.